.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -31,22 +31,10 @@ Steps to reproduce the behavior:
 
 **Version information**
 <!-- please fill out this information below completely, it will help us solve your issue faster -->
```

Comment:
Nitpick: can these be proper sentences?

"Please fill out this information completely. It will help us solve your issue faster."